### ISSUE TITLE
Add a Dropdown box for Adding Pokemon to Entree Forest

### ISSUE THREAD

**Issue by ScarlettHaze** [kuroppoi/entralinked#9]
Add a Dropdown box for Adding Pokemon to Entree Forest, It makes it easier to add Pokemon that were legally released via Dream World. 
The lists are in here:
https://github.com/suloku/BW_tool/blob/master/BW_tool/Editors/Entralink_DreamWorld.cs

**Comment by kuroppoi** [comment]
I got rid of all the magic number inputs and replaced them with drop-down lists, but legality is something that users will have to look into themselves for the time being.

**Comment by kuroppoi** [comment]
Reopening tthis as I'd like to work on a legal-only filter now

**Comment by ScarlettHaze** [comment]
Here is a list of the usable unused Pokemon they have sprites in the game so they appear https://tcrf.net/Pok%C3%A9mon_Black_and_White/Unused_Pok%C3%A9mon_Data#Unused_Dream_World_Data

**Comment by kuroppoi** [comment]
> Here is a list of the usable unused Pokemon they have sprites in the game so they appear https://tcrf.net/Pok%C3%A9mon_Black_and_White/Unused_Pok%C3%A9mon_Data#Unused_Dream_World_Data

Yup, those are implemented already along with all the B2W2 exclusives.

**Comment by ashgx** [comment]
May I ask if this will be updated in the future？

**Comment by kuroppoi** [comment]
It's being worked on :)
